src/NzbDrone.Core/Notifications/MediaBrowser/MediaBrowserSettings.cs

```diff
@@ -43,6 +45,15 @@ public MediaBrowserSettings()
         [FieldDefinition(5, Label = "Update Library", HelpText = "Update Library on Import, Rename, or Delete?", Type = FieldType.Checkbox)]
         public bool UpdateLibrary { get; set; }
 
+        [FieldDefinition(5, Label = "Update Library By Name", HelpText = "Update Library by name rather than path(Requires 'UpdateLibrary')", Type = FieldType.Checkbox, Advanced = true)]
```

Comment:
It's not just name anymore, but we don't provide this option elsewhere we should just assume that it's wanted and update an de-duped paths found by ID or name as well as anything that is remapped.

Reply:
If I understand correctly, you would want to automatically just update every possible option and let Jellyfin figure out whether there's actually anything to do? That seems a bit wasteful perhaps. I don't think this is obscure enough as that having it would be confusing.

Reply:
That's what we do with Kodi and previously with Plex (before a lot of people changed away from TheTVDB metadata in Plex), we smartly find the matching series and update that instead of updating nothing because the paths don't match before falling back to everything.

The vast majority of people are going to have a single library where that series exists, Sonarr will find the matching series and update the path that Jellyfin reports. The only thing wasted would be an API request to try and find the matching series.

If the user has set up a mapping of paths and Sonarr finds a path from JF and they're the same then it's still only one path being updated, if they happen to be different then there would be multiple.